packages/dds/tree/src/test/shared-tree-core/utils.ts

```diff
@@ -310,3 +314,19 @@ export class TestSharedTreeCore extends SharedObject {
 		return this.kernel.getEditor();
 	}
 }
+
+export class TestChangeEnricher implements ChangeEnricherReadonlyCheckout<TestChange> {
+	public updateChangeEnrichments(change: TestChange, revision: RevisionTag): TestChange {
+		if (TestChange.isNonEmptyChange(change)) {
+			return {
+				inputContext: change.inputContext.map((i) => i * 1000),
```

Comment:
What's the * 1000 do here?

Reply:
It's an arbitrary way of affecting the changeset to make it clear that it has been enriched.